src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -413,20 +422,21 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin, OrgPersonMix
   @Getter isFirmChangeFiling!: boolean
   @Getter isFirmConversionFiling!: boolean
   @Getter isFirmCorrectionFiling!: boolean
+  @Getter isLimitedExtendRestorationFiling!: boolean
+  @Getter isLimitedConversionRestorationFiling!: boolean
   @Getter isRoleStaff!: boolean
   @Getter hideChangeButtonForSoleProps!: boolean
 
```

Comment:
@severinbeauvais, somehow I do not see your comment captured in screenshot below. The change was to not add ResourceIF in this component and instead use prop and let parent use resource file to ge t the value

![image](https://user-images.githubusercontent.com/116035339/224786352-3a2a35d1-dd33-46cf-8d96-e0b0fa8bc875.png)

Reply:
This component looks good to me. You can resolve this conversation.